TWLight/static/css/new-local.css

```diff
@@ -1345,6 +1345,31 @@ ul.timeline-home li {
 	display:none;
 }
 
+.partner-detail-logo{
+    position: relative;
+    padding: 0 10% 0 10%;
+    margin: 0 auto 2em auto;
+    max-height: 7em;
+    min-height: 3em;
+    z-index: 1;
+}
+
+.resource-label-subject{
+    background-image: url('../img/bullet-subject.png');
+    background-repeat: no-repeat;
+    background-position: top left;
+    min-height: 2em;
+    padding: 6px 0px 0px 25px;
```

Comment:
For me, the background image for this and the languages butt up against the text
![image](https://user-images.githubusercontent.com/2986893/145103244-ab476bb0-9a2a-43aa-a006-1f9f92884f1e.png)

increasing that padding from 25 to 30 px helps out.
![image](https://user-images.githubusercontent.com/2986893/145103155-e8ae7051-9a5a-47ef-8661-ea0277b33073.png)

Reply:
Good catch!